src/traces/parcoords/defaults.js

```diff
@@ -17,9 +17,9 @@ var maxDimensionCount = require('./constants').maxDimensionCount;
 function handleLineDefaults(traceIn, traceOut, defaultColor, layout, coerce) {
 
     coerce('line.color', defaultColor);
-    coerce('line.colorscale');
 
-    if(hasColorscale(traceIn, 'line')) {
+    if(hasColorscale(traceIn, 'line') && Lib.isArray(traceIn.line.color)) {
+        coerce('line.colorscale');
         colorscaleDefaults(traceIn, traceOut, layout, coerce, {prefix: 'line.', cLetter: 'c'});
```

Comment:
There's no point in attempting to use a color scale if `line.color` is not an array but e.g. a plain color, because the `line.color` numerical array values are mapped to the colors on the color scale.
Also, now `line.colorscale` is only performed if it's warranted.